File: DailyFlash Solutions/DailyFlash Solutions/01-DailyFlash_Solutions/14_Jan_Solutions_Two/java/CheckAlphabet.java

/*

Program 1: Write a Program to check whether the Character is Alphabet or
not.
Input: v
Output: v is an alphabet.

*/


class CheckAlphabet{

	
	public static void main(String args[]){
		
		char ch = '*';

        	if((ch>=65&&ch<=90)||(ch>=97&&ch<=122))
                	System.out.printf("%c is an alphabet",ch);
        	else
                	System.out.printf("%c is not an alphabet",ch);

	}	

}	

File: DailyFlash Solutions/DailyFlash Solutions/01-DailyFlash_Solutions/14_Jan_Solutions_Two/c/checkAlphabet.c
/*
 
Program 1: Write a Program to check whether the Character is Alphabet or not.
Input: v
Output: v is an alphabet.


 */

#include<stdio.h>

void main(){

	char ch = '*';
	
	if((ch>=65&&ch<=90)||(ch>=97&&ch<=122))
		printf("%c is an alphabet",ch);
	else
		printf("%c is not an alphabet",ch);	

}	
